README.md

```diff
@@ -110,3 +164,7 @@ You can test your installation by running  `pytest -v`.
 
```

Comment:
It would be good to have a section on contributing: 
- Our python style.
- Mypy.
- Expected test coverage.